packages/core/src/controllers/relayer.ts

```diff
@@ -88,12 +89,20 @@ export class Relayer extends IRelayer {
   public async init() {
     this.logger.trace(`Initialized`);
     await this.createProvider();
-    await Promise.all([this.messages.init(), this.transportOpen(), this.subscriber.init()]);
+    await Promise.all([this.messages.init(), this.subscriber.init()]);
+    try {
+      await this.transportOpen();
```

Comment:
what kind of errors will this catch? all? Ideally failover only kicks in for errors lower than HTTP

Reply:
This will catch if:
* `provider.connect()` (i.e. `websocket.open()`) fails
* the `transportOpen` logic calls rejectTransport after 5 seconds due to presumably stalled connection attempt